_data/navigation.yml

```diff
@@ -87,25 +87,23 @@ docs_integrate:
     children:
       - title: "Establish a Relationship to another Identity"
         url: integrate/establish-a-relationship-to-another-identity
-  - title: Working with requests
+  - title: Working with Requests
```

Comment:
If we keep this category, we'll again have an -ing form. Maybe we should stick with "Manage Requests" or we can come up with something new without -ing. @britsta What do you think?

Reply:
Work with Requests?

Reply:
Genius 👍